nmz/cli/inspectors/proc.go

```diff
@@ -44,6 +46,7 @@ func init() {
 	procFlagset.IntVar(&_procFlags.RootPID, "pid", -1, "PID for the target process tree")
 	procFlagset.DurationVar(&_procFlags.WatchInterval, "watch-interval", 1*time.Second, "Watching interval")
 	procFlagset.StringVar(&_procFlags.Cmd, "cmd", "", "Command for target process")
+	procFlagset.StringVar(&_procFlags.Stdout, "stdout", "", "Stdout for target process (used if -cmd option is given)")
```

Comment:
I think `-cmd-stdout` is much more descriptive.
If you feel `-cmd-stdout` is too long, I'm +1 for keeping `-stdout`